### PR TITLE
Ready plugin with all settings and grunt setup

### PR DESCRIPTION
**Pull request by imnavanath**
- Corrected previous code for wp_head and wp_footer actions
- Done with grunt setup (pot generation, readme.md file generation)
- Fixed 4 reported issues
- Removed class_exists condition check
- Removed empty callback function
- Code is intended properly

### CODE REVIEW

File: classes/class-nwcc-loader.php [hunk 1]
```diff
@@ -0,0 +1,358 @@
+<?php
+/**
+ * NWCC Loader.
+ *
+ * @package NWCC
+ */
+if( ! class_exists( 'Multisite_Script_Class' ) ) {
```
Comment (patilvikasj) [inline classes/class-nwcc-loader.php]: Remove this class_exists check.

File: classes/class-nwcc-loader.php [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+            switch_to_blog( $this->current_blog );
+
+            // Add required actions.
+            add_action( 'network_admin_menu', array( $this, 'add_settings_page' ), 9999 );
+            add_action( 'admin_init', array( $this, 'register_settings' ) );
+            add_action( 'init', array( $this, 'admin_post_edit_options' ) );
+            add_action( 'init', array( $this, 'run_loader' ) );
+
+            // Load textdomain translations.
+            $this->load_textdomain();
```
Comment (patilvikasj) [inline classes/class-nwcc-loader.php]: Need to correct indentation for this block of code.

File: classes/class-nwcc-loader.php [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+	    /**
+		 * Function Name: print_section_info
+		 * Function Description: Prints information about the section
+		 *
+		 * @since  1.0.0
+		 * @return void
+		 */
+	    public function print_section_info() {
+	        //Nothing to do here.
+	    }
```
Comment (patilvikasj) [inline classes/class-nwcc-loader.php]: Remove empty functions.

File: classes/class-nwcc-loader.php (hunk 1 above)
Comment (imnavanath) [reply classes/class-nwcc-loader.php]: Removed condition

File: classes/class-nwcc-loader.php (hunk 2 above)
Comment (imnavanath) [reply classes/class-nwcc-loader.php]: Code is correctly intended here - https://github.com/imnavanath/network-wide-custom-code/blob/master/classes/class-nwcc-loader.php

File: classes/class-nwcc-loader.php (hunk 3 above)
Comment (imnavanath) [reply classes/class-nwcc-loader.php]: Removed the empty function

**Pull request closed without merging**